marimo/_cli/sandbox.py

```diff
@@ -144,8 +182,14 @@ def run_in_sandbox(
         # which may conflict with the sandbox requirements
         "--no-project",
         "--with-requirements",
-        temp_file_path,
-    ] + cmd
+        dep_temp_file_path,
+    ]
+
+    if local_editable_packages:
+        cmd.append("--with-editable")
```

Comment:
whats the different between passing these to `--with-editable` vs putting them in the requirements.txt as `-e /path/to/editable`

Reply:
I ran into an error that said literally `-` is not expected to begin the dependency name. Maybe that's worth exploring an unblock natively in one requirements file, will attempt later. Could be a uv inconsistency with real pip.